data_architect_misc/data_transformer/transform_functions/na_united_states_transform_functions.py

```diff
@@ -0,0 +1,61 @@
+"""This is the subclass of Transform function for United States (NA division).
+
+We will define transform functions specific to United States here.
+
+Author: Maicol Contreras
+Last Modified: November 24, 2020
+"""
+
+import pandas as pd
+import re
+import numpy as np
+from time import strptime
+
+from constants import comp_harm_constants
+from constants.transform_constants import KEY_CURRENT_INPUT_FILE, KEY_DELIMITER, KEY_HEADER
+from transform_functions.common_comp_harm_transform_functions import CommonCompHarmTransformFunctions
+
+
+class NaUnitedStatesTransformFunctions(CommonCompHarmTransformFunctions):
+    """
+    All custom (uncommon) transform functions **SPECIFIC to
+    individual processing task** must be defined as part
+    of this class.
+    """
+    UNITED_STATES_SPECIFIC_CATEGORY_MAPPINGS = {
+        # Add any United States-specific mappings that cannot be used for other NA countries here.
+    }
+
+    def __init__(self, config):
+        self.config = config
+
+    def create_a_new_date_column_based_taking_the_date_from_the_TOTAL_column(self,
```

Comment:
Can you please explain me what this method is doing? An example file or copy-paste of a few data rows would be great.

From reading the method name, I am thinking something like this -- in the 'Total' column, you have some date values. You extract them and assign them to the 'Date' column? 

Thank you.